jsonserver.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
what is this file for?

Reply:
It file created with json-server . I use it for mock server

Reply:
deleted